app/task/Install.php

```diff
@@ -0,0 +1,173 @@
+<?php
+
+/*
+  +------------------------------------------------------------------------+
+  | Phosphorum                                                             |
+  +------------------------------------------------------------------------+
+  | Copyright (c) 2013-present Phalcon Team (https://www.phalconphp.com)   |
+  +------------------------------------------------------------------------+
+  | This source file is subject to the New BSD License that is bundled     |
+  | with this package in the file LICENSE.txt.                             |
+  |                                                                        |
+  | If you did not receive a copy of the license and are unable to         |
+  | obtain it through the world-wide-web, please send an email             |
+  | to license@phalconphp.com so we can send you a copy immediately.       |
+  +------------------------------------------------------------------------+
+*/
+
+namespace Phosphorum\Task;
+
+use Phalcon\Di;
+use League\CLImate\CLImate;
+use Phosphorum\Console\AbstractTask;
+use Phosphorum\Console\OptionParser;
+use Phosphorum\Exception\InvalidParameterException;
+
+/**
+ * Phosphorum\Task\Install
+ *
+ * @package Phosphorum\Task
+ */
+class Install extends AbstractTask
+{
+    protected $pathesList = [
+        'assets_path',
+        'log_path',
+        'pids_path',
+        'annotations_cache',
+        'config_cache',
+        'data_cache',
+        'metaData_cache',
+        'views_cache',
+        'volt_cache',
+    ];
+
+    protected $user = [
+        'userName' => 'www-data',
+    ];
+
+    /**
+     * @Doc("Check folders that must have permission to write files into them")
+     */
+    public function checkDirs()
+    {
+        if (!$this->assertNotWindows()) {
+            (new CLImate())->info("This command can't check writing permission on WINDOWS");
+        }
+
+        $this->setUserInfo($this->getCommand());
+        $this->checkPaths($this->getPaths());
+    }
+
+    /**
+     * @return bool
+     */
+    protected function assertNotWindows()
+    {
+        if (strtoupper(substr(PHP_OS, 0, 3)) === 'WIN') {
+            return false;
+        }
+
+        return true;
+    }
+
+    /**
+     * @return array
+     */
+    protected function getCommand()
+    {
+        return OptionParser::parse($_SERVER['argv']);
+    }
+
+    /**
+     * @param array $command
+     * @return void
+     * @thrown InvalidParameterException
+     */
+    public function setUserInfo($command)
+    {
+        if (isset($command['name']) && !empty($command['name'])) {
+            $this->user['userName'] = $command['name'];
+        }
+
+        if (!$this->user = posix_getpwnam($this->user['userName'])) {
```

Comment:
To properly get the info about a user test if `function_exists('posix_getpwnam')`. Are you sure about Windows? I'm assume no.